### ISSUE TITLE
investigate a flag (or feature-test) to use the actual `let` keyword instead of try/catch

### ISSUE THREAD

**Issue by getify** [getify/let-er#3]
ES6 is adding the `let` keyword, which will allow block-scoped variable declarations like so:

``` js
if (something) {
   let isTrue = something;
   console.log(isTrue);
}
isTrue; // ReferenceError
```

Unfortunately, the better `let (foo) { .. }` style of code (which btw works in FF since forever), is not supported, which is why BlockScoper will be useful not only today but into the future.

This code transpiler currently makes the try/catch rewrite for let blocks, but it would probably be nice to have it be able to use the new `let` keyword in the future once it arrives via ES6.

For instance, we can use naked blocks, and use the style I suggested here: https://gist.github.com/getify/5285514

So, the code transpilation for `let (foo) { .. }` could just be in ES6:

``` js
/*let*/{ let foo;
  // ..
}
```

Need to investigate whether this would just be a flag you used to switch between try/catch and `let` declarations, or if there's anyway feasible to have a runtime feature-test for `let` to automatically switch. Probably not, but worth exploring briefly.

**Comment by getify** [comment]
My guess is this will probably land as a flag to control the transpilation output. The reason is, you could use _let-er_ to transform the non-standard `let (..) { .. }` blocks that _let-er_ handles into `{ let ..; .. }` code, and THEN pass that code through something like traceur, which would catch all the `let`s, not just the ones that _let-er_ will produce.'

That flag will even make _let-er_ more useful even after ES5 has gone away and you're just transpiling from ES6++ to ES6.

**Comment by getify** [comment]
This landed as of version 0.1.0-a. https://github.com/getify/let-er#options
